network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/featured_blogs.html

```diff
@@ -18,48 +18,60 @@ <h4 class="h4-heading">{% trans "Latest news" %}</h4>
     <div class="row">
       <div class="col-12 full-bleed-xs">
         <div class="key-item mx-2 mx-md-4 p-3">
-          <p class="h6-heading mb-2">
-            {% for category in blog.blog.specific.category.all %}
-            <a href="/blog/category/{{category.slug}}">{{ category }}</a>{% if not forloop.last %}, {% endif %}
-            {% endfor %}
-          </p>
           <h5 class="mb-2 h3-heading">
             <a href="{{blog.blog.url}}">{{blog.blog.title}}</a>
           </h5>
-          <p class="body-small mb-2">
-            {% if blog.blog.author %}
-              {% blocktrans with author=blog.blog.author %}by {{author}}{% endblocktrans %}
-            {% endif %}
-            {% if blog.blog.first_published_at %} {% blocktrans with date=blog.blog.first_published_at|date:"DATE_FORMAT" %}on {{date}}{% endblocktrans %}{% else %}not published yet{% endif %}
-          </p>
-          <p>{{blog.blog.search_description}}</p>
         </div>
       </div>
     </div>
   </div>
   {% endif %}
   {% endfor %}
 
+
   {% if featured_blogs|length > 1 %}
   <div class="col-md-6 mb-0 pb-4">
     {% for blog in featured_blogs %}
+    {% get_authors blog as authors %}
```

Comment:
I'd like to not use a tag here if the only thing the tag does is repack the data to omit a level of indirection. Let's stick with ` {% for entry in blog.authors.all %}` and then pull out `entry.author.image` etc. - that may look uglier, but it's less magic, and less magic when it comes to Django means it's infinitely more accessible when it comes time to figure out a bug for folks less familiar with Django/wagtail in the future.

Reply:
The reason I added the template tag was because we're traversing from Home Page -> Blog Page Orderable -> Blog Page -> Blog Authors. 

Here's the original work before the template tag: https://github.com/mozilla/foundation.mozilla.org/pull/4957/commits/680ed8de3dcba008056fa42b006dd6b2c3221383#diff-084370aab012384922f2ae8b4676e3d2R39-R62

Though if you feel strongly about this I can revert this work easily. Let me know what your final thoughts on this are.

Reply:
Fair, let's leave it a tag then, but let's rename it to `get_all_authors`. Small change, but makes it a little clearer that this is a custom function and a magical Django function `get_[fieldname]`.

Reply:
```suggestion
    {% get_all_authors blog as authors %}
```